software_reseller/data/ir_actions_act_window.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="software_reseller.all_licenses_ce314e8b-2439-473a-b7a7-493af7707108" model="ir.actions.act_window" context="{'studio': True}">
+        <field name="domain">[('project_id', 'ilike','%licenses%')]</field>
```

Comment:
Why no boolean field?

Reply:
?

Reply:
name matching does not seem like a good idea...

Reply:
it is easier to do name matching for a domain, as it can easily be adapted by the customer in his database, moreover the projects matching the name are created in the data. Custom fields, add more complexity for just a feature of 'filtering'